MvvmCross/Platforms/Wpf/Presenters/MvxWpfExtensionMethods.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Linq;
+using MvvmCross.Platforms.Wpf.Presenters.Attributes;
+
+namespace MvvmCross.Platforms.Wpf.Presenters
+{
+    public static class MvxWpfExtensionMethods
```

Comment:
It seems this only contains Presenter extensions. Please name it `MvxWpfPresenterExtensions`